src/content_mapblock.cpp

```diff
@@ -595,6 +602,24 @@ void MapblockMeshGenerator::drawLiquidTop()
 	collector->append(tile_liquid_top, vertices, 4, quad_indices, 6);
 }
 
+void MapblockMeshGenerator::drawLiquidBottom()
+{
+	video::S3DVertex vertices[4] = {
+		video::S3DVertex(-BS / 2,  -BS / 2, -BS / 2, 0, 0, 0, color_liquid_top, 0, 0),
+		video::S3DVertex( BS / 2,  -BS / 2, -BS / 2, 0, 0, 0, color_liquid_top, 1, 0),
+		video::S3DVertex( BS / 2,  -BS / 2,  BS / 2, 0, 0, 0, color_liquid_top, 1, 1),
```

Comment:
Trivial double space in the 2nd argument

Reply:
Fixed.